### DISCUSSION TITLE
Welcome to assembler-simulator Discussions!

### DISCUSSION THREAD

**Discussion by exuanbo** [exuanbo/assembler-simulator#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by PhilippKonnak** [comment]
Hey,
I hope that you might help me with following issue.
My task is to simulate a direct mapped cache for x86 Assembler.
I wrote assembler code.
But I struggle at an actually "easy" issue.
I have 2 Integers in a 32 Bit register. 
I put 32 (over an input) into eax and I put 4 into ebx (over an input). Now I want to divide it. 32/4 = 8.
So what I wrote is:

mov eax, [esp+8]; Printing this line will show me 32. Thats fine.
mov ebx, [esp+4]; Printing this line will show me 4. Thats fine, too.
div ebx; (Line 86 will cause the problem now)

My terminal returns the following: "Floating Point exception"

**Reply by exuanbo** [reply]
I'm sorry, but I don't think I'm the best person to help with your specific problem.

**Comment by neel04** [comment]
Can we get vim editor in there please? 🙏

**Reply by exuanbo** [reply]
Adding Vim keybindings is simple, but first I need to think about where to put the options related to the editor.

**Reply by exuanbo** [reply]
Created an issue for tracking https://github.com/exuanbo/assembler-simulator/issues/27

**Reply by neel04** [reply]
Perhaps just a toggle-able Vim logo in the upper-right corner, so only noticed by those who're looking for it?

**Reply by exuanbo** [reply]
After reloading the app from the update prompt, you will now find an option in the `Configuration` menu. Enjoy :)

**Reply by neel04** [reply]
What a legend! thank you so so much! 🔥
